### PR TITLE
Fix web3 build error

### PR DESCRIPTION
**Pull request by xin-hedera**
**Description**:

This PR fixes the web3 module build error.

- force using swirlds-common:0.31.0

**Related issue(s)**:

Relates to #4704 

**Notes for reviewer**:

Workaround, once the dependency issue is solved in hedera-evm-api artifact, we should remove the hardcoded version

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by edwin-greene — APPROVED** [review]
Looks good

File: hedera-mirror-web3/build.gradle.kts [hunk 1]
```diff
@@ -27,7 +27,10 @@ plugins {
 dependencies {
     implementation(platform("org.springframework.cloud:spring-cloud-dependencies"))
     implementation(project(":common"))
-    implementation("com.hedera.evm:hedera-evm-api")
+    implementation("com.hedera.evm:hedera-evm-api") {
+        exclude("com.swirlds", "swirlds-common")
```
Comment (steven-sheehy) [inline hedera-mirror-web3/build.gradle.kts:31]: Remove the exclude, it's not necessary.

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: hedera-mirror-web3/build.gradle.kts [hunk 2]
```diff
@@ -27,7 +27,10 @@ plugins {
...
+    }
+    implementation("com.swirlds:swirlds-common:0.31.0")
```
Comment (steven-sheehy) [inline hedera-mirror-web3/build.gradle.kts:33]: Can move this to root build.gradle.kts.

Comment (steven-sheehy) [inline hedera-mirror-web3/build.gradle.kts:33]: Add a comment to both dependencies that this is a temporary transient dependency

**Comment by steven-sheehy** [comment]
Also please bump SDK to 2.18.0 in gradle to fix build.

**Review by steven-sheehy — APPROVED** [review]

**Pull request merged**